webview/src/setup/components/App.tsx

```diff
@@ -19,6 +20,9 @@ export const App: React.FC = () => {
   const [cliCompatible, setCliCompatible] = useState<boolean | undefined>(
     undefined
   )
+  const [dvcCliDetails, setDvcCliDetails] = useState<DvcCliDetails | undefined>(
```

Comment:
Currently were passing around an object, but we could also just have two new properties, `dvcVersion` and `dvcExampleCommand`.